e2e-tests/insurance/cypress/e2e/journeys/policy/change-your-answers/broker/change-your-answers-change-address-not-based-in-uk-to-based-in-uk-multiple-addresses-available.spec.js

```diff
@@ -0,0 +1,106 @@
+import { summaryList } from '../../../../../../../pages/shared';
+import { brokerConfirmAddressPage } from '../../../../../../../pages/insurance/policy';
+import { EXPECTED_UNDERGROUND_STATION_SINGLE_LINE_STRING } from '../../../../../../../constants';
+import { POLICY as POLICY_FIELD_IDS } from '../../../../../../../constants/field-ids/insurance/policy';
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import checkSummaryList from '../../../../../../../commands/insurance/check-policy-summary-list';
+
+const {
+  USING_BROKER,
+  BROKER_DETAILS: { NAME, EMAIL },
+  BROKER_ADDRESSES: { SELECT_THE_ADDRESS },
+  BROKER_MANUAL_ADDRESS: { FULL_ADDRESS },
+} = POLICY_FIELD_IDS;
+
+const {
+  ROOT,
+  POLICY: { BROKER_ADDRESSES_CHANGE, BROKER_CONFIRM_ADDRESS_CHANGE, CHECK_YOUR_ANSWERS },
+} = INSURANCE_ROUTES;
+
+/**
+ * TODO
```

Comment:
Multiple TODOs, I suppose this will be removed after the PR is merged?

Reply:
PR is WIP until another PR is merged. These will be resolved once the blocking PR is merged.